config/routes.rb

```diff
@@ -186,6 +186,11 @@
       resources :partnerships, only: :index
       resource :programme, only: %i[edit], controller: "choose_programme"
 
+      resources :participants, only: %i[index show update], constraints: ->(_request) { FeatureFlag.active?(:induction_tutor_manage_participants) } do
+        get :edit_details, path: "edit-details"
+        get :edit_mentor, path: "edit-mentor"
+      end
```

Comment:
`constraints` are fancy, but really, we could also brute force it with 
```
if FeatureFlag.active?(:induction_tutor_manage_participants)
  resources :participants ...
end
```
The fact that you aren't using `request` kind of tells me `constraints` was designed to be a little more sophisticated.

Reply:
@kicferk1 - Constraint block is eveluated per each reuqest, while hardcoded condition will be run once on application start. The latter requires application restart when the flag is enabled.

Reply:
originally did it like that, but it meant that the server required a reboot after updating a feature flag otherwise a bunch of pages would error - spoke about it with ethan and went with this approach instead

Reply:
Ah, good point, we won't do a re-stage when changing feature flags :)